src/main/java/core/basesyntax/JavaDateTimeApi.java

```diff
@@ -19,7 +37,16 @@
      *                 В любом другом случае бросить DateTimeException
      **/
     public String todayDate(DateTimePart datePart) {
-        return "Today";
+        Map<DateTimePart, String> dateRequest = new HashMap<>();
+        dateRequest.put(DateTimePart.FULL, localDateNow.toString());
+        dateRequest.put(DateTimePart.YEAR, String.valueOf(localDateNow.getYear()));
+        dateRequest.put(DateTimePart.MONTH, String.valueOf(localDateNow.getMonth()));
+        dateRequest.put(DateTimePart.DAY, String.valueOf(localDateNow.getDayOfYear()));
```

Comment:
good, but if you want more briefly code, you can use operator switch(){case} without Map

Reply:
I know but wanted to try to use Map:). Don't think Switch is shorter.